metadata-ingestion/src/datahub/ingestion/source/mongodb.py

```diff
@@ -90,6 +103,10 @@ class MongoDBConfig(PlatformInstanceConfigMixin, EnvConfigMixin):
     # mongodb only supports 16MB as max size for documents. However, if we try to retrieve a larger document it
     # errors out with "16793600" as the maximum size supported.
     maxDocumentSize: Optional[PositiveInt] = Field(default=16793600, description="")
+    ingest_data_platform_instance_aspect: Optional[bool] = Field(
+        default=False,
```

Comment:
we probably want this to be default true, or at least done automatically if platform_instance is set. don't think it makes sense to make it a config option